selfdrive/car/honda/carstate.py

```diff
@@ -10,7 +10,7 @@
 TransmissionType = car.CarParams.TransmissionType
 
 
-def get_can_signals(CP, gearbox_msg="GEARBOX"):
+def get_can_signals(CP, *, gearbox_msg):
```

Comment:
why change this?

Reply:
This is a remnant from abstracting the pedal message, but I suggest keeping it since there are further opportunities for abstracting messages (e.g. `SCM_FEEDBACK` vs. `SCM_BUTTONS` for `MAIN_ON` signal).

  * this function only has one caller, and the arg default isn't used-- avoid maintaining the `GEARBOX` string literal in two places
  * when additional string args are added in the future, there's potential to mix them up and cause silent bugs rather than runtime exceptions-- safer to burden the single caller with keyword args

Reply:
hmm, made it a non kw arg